style.css

```diff
@@ -0,0 +1,359 @@
+  width: 375px;
+  height: 718px;
+  border: 1px solid #dfe1e6;
+  border-bottom-left-radius: 70px;
+  background-color: #fff;
+  padding-top: 290px;
+  padding-right: 30px;
+  padding-left: 23px;
+  padding-bottom: 204px;
+}
+
+p {
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+}
+
+.headline-section div {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+}
+
+h1 {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 50px;
+  color: #172b4d;
+  padding-right: 10%;
+  margin-bottom: 0;
+}
+
+.about-me {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.connect {
+  color: #7f8cff;
+  font-weight: 500;
+  font-size: 15px;
+  margin-block-end: 1em;
+}
+
+.social-icons {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding-left: 0;
+  height: 22px;
+  width: 22px;
+  padding-right: 15px;
+  gap: 20px;
+  margin-top: 4px;
+  margin-block-end: 5em;
+}
+
+#projects {
+  background-color: #f7f7f9;
+  width: 375px;
+  display: grid;
+  grid-template-columns: auto;
+  justify-content: center;
+}
+
+.project {
+  padding: 1% 2%;
+  background-color: #fff;
+  margin-top: 30px;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+}
+
+.project-image {
+  display: flex;
+  justify-content: center;
+  padding-left: 15px;
+  padding-right: 15px;
+  padding-top: 15px;
+}
+
+.project-content {
+  padding-left: 15px;
+  padding-right: 15px;
+}
+
+.project-heading {
+  color: #172b4d;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  margin-bottom: 10px;
+  margin-top: 10px;
+}
+
+.content-text {
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  color: #7a869a;
+  display: inline;
+}
+
+.canopy {
+  color: #344563;
+}
+
+.count-vector {
+  padding-left: 5px;
+  padding-right: 5px;
+}
+
+.description-text {
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  padding-right: 19px;
+  color: #344563;
+}
+
+.language-used {
+  background-color: #ebebff;
+  color: #6070ff;
+  padding: 4px 12px 4px 12px;
+  font-size: 12px;
+  border-radius: 8px 8px 8px 8px;
+}
+
+.project-link {
+  color: #396df2;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+}
+
+.btn {
+  padding: 12px 12px 12px 12px;
+  margin-top: 25px;
+  margin-bottom: 15px;
+  background-color: white;
+  border: 1px solid #396df2;
+  border-radius: 8px 8px 8px 8px;
+}
+
+.btn:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+.btn:active {
+  background-color: #2230d2;
+  color: white;
+}
+
+#second-project {
+  margin-top: 100px;
+}
+
+#third-project {
+  margin-top: 100px;
+}
+
+#fourth-project {
+  margin-top: 100px;
+  margin-bottom: 100px;
+}
+
+#about-section {
+  background-color: white;
+  border-top-right-radius: 80px;
+  width: 375px;
+}
+
+.about {
+  padding-right: 29px;
+  padding-left: 29px;
+  padding-top: 80px;
+  padding-bottom: 50px;
+  border-top-right-radius: 50px;
+}
```

Comment:
- [ ] I think it would be better to use `display: flex` to align the content of the about section as well. This is because `display: flex` makes it easier to design flexible responsive layout structure 🛠 This will help you in the upcoming milestones as well. Please consider fixing this as well. I know you can do that 💪🥇